repos.go

```diff
@@ -298,6 +304,40 @@ func repo(owner, name string) Repo {
 	}
 }
 
+func repoRecentReleases(owner, name string, count int) []Release {
+	var releases []Release
```

Comment:
linter just yelled at me for the same thing 😂 
```suggestion
	releases  := make([]Release, count)
```

Reply:
beware though, the api might not actually reply with that many repos... 

probably better to move this line a couple of lines down and do

```go
releases := make([]Release, len(repoRecentReleasesQuery.Repository.Releases.Nodes))
```

Reply:
> beware though, the api might not actually reply with that many repos...
You mean if they choose to ignore that value on their end? so still worth checking 🤔

Reply:
I mean, if the user ask for 10 releases, but the repo only have 3, the array will still have length=10

Reply:
Ah of course! Thanks Carlos :)